### PR TITLE
Feature/js save idea

### PR DESCRIPTION
**Pull request by Romeslayer**
PR Template (GitHub PR Comments):
What (if any) features are you implementing?
The form section has functionality for the save button. The form uses the information from the body and title inputs to create a new instance of an Idea, saves that Idea to an array and creates a new Idea class on the DOM.
What (if anything) did you refactor?
n/a
Were there any issues that arose?
n/a
Is there anything that you need from your teammate?
n/a
Any other comments, questions, or concerns?
n/a

### CODE REVIEW

**Comment by userigorgithub** [comment]
Amazing job DeNaje and Oakley.

File: main.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            <div class="card-top">
+              <img src="./assets/star.svg" alt="star"/>
+              <img src="./assets/delete.svg" alt="delete"/>
+            </div>
+            <div class="card-body">
+              <h4>${element.title}</h4>
+              <p>${element.body}</p>
+            </div>
+            <div class='card-comment'>
+              <img src="./assets/comment.svg" alt="comment"/>
```
Comment (userigorgithub) [inline main.js:52]: Probably should be same line, src and alt.

**Review by userigorgithub — APPROVED** [review]
Looks great, seems like iteration 2 is done.

**Review by oakleywindiate — APPROVED** [review]
This looks good. Before submitting final let's remove some extra lines and review CSS styling.

**Pull request merged**